dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/controller/AckFromDrcController.java

```diff
@@ -41,14 +40,6 @@ public class AckFromDrcController {
     @ApiResponse(responseCode = "200",
             content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
                     schema = @Schema(implementation = String.class)))
-    @ApiResponse(responseCode = "400",
```

Comment:
I think you might want just one example of a failure (e.g. `404` if you provide an non-existing contribution id) and the fact that we respond with the [`ProblemDetail`](https://docs.spring.io/spring-framework/reference/web/webmvc/mvc-ann-rest-exceptions.html) class now.

You could almost certainly just use old lines 44-47 (deleted here) but with the standard Spring class `ProblemDetail.class` instead of our own homegrown `ErrorResponse.class`.

Reply:
OK, adding one example for 404 as suggested.